roles/kubernetes-apps/ansible/templates/kubedns-deploy.yml

```diff
@@ -38,7 +38,7 @@ spec:
         livenessProbe:
           httpGet:
             path: /healthz
-            port: 8080
+            port: {{ kube_apiserver_insecure_port }}
```

Comment:
Right, so https://github.com/kubernetes-incubator/kargo/blob/master/roles/kubernetes/master/handlers/main.yml#L42 should be changed. The template you're updating here is for kubedns, not apiserver. 
There are many kubernetes apps that expose a `/healthz` endpoint, it's not limited to only the apiserver.